src/schema/schema_constants.py

```diff
@@ -14,3 +15,13 @@ class SchemaConstants(object):
     ACCESS_LEVEL_PUBLIC = 'public'
     ACCESS_LEVEL_CONSORTIUM = 'consortium'
     ACCESS_LEVEL_PROTECTED = 'protected'
+
+    DOI_BASE_URL = 'https://doi.org/'
+
+# Define an enumeration to classify an entity's visibility, which can be combined with
+# authorization info when verify operations on a request.
+class DataVisibilityEnum(Enum):
+    PUBLIC = SchemaConstants.ACCESS_LEVEL_PUBLIC
+    # Since initial release just requires public/non-public, add
+    # another entry indicating non-public.
+    NONPUBLIC = 'nonpublic'
```

Comment:
What is the purpose of 'nonpublic' access level?

Reply:
The public/non-public notion of these constants is the thumbs up or thumbs down on whether requested data can be shown to non-authenticated users.  This is a little different from the data_access_level, which is one component of deciding if something is public/non-public, determined by app.py's `_get_entity_visibility()`.

For example, the criteria for a Collection to be seen are numerous, including looping over each Dataset it contains:
[https://github.com/sennetconsortium/entity-api/blob/f98e2404e0bc2b10e5b23718b31c6fdb54017732/src/app.py#L465-L480](https://github.com/sennetconsortium/entity-api/blob/f98e2404e0bc2b10e5b23718b31c6fdb54017732/src/app.py#L465-L480)